pkgs/applications/networking/p2p/pyrosimple/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, python3Packages
+, nix-update-script
+, pyrosimple
+, testers
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "pyrosimple";
+  version = "2.7.0";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-SMqzvTbWFHwnbMQ+6K0m1v+PybceQK5EHEuN8FB6SaU=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
```

Comment:
move these into the function args instead of `python3Packages`, in conjunction with the all-packages change below